Integrations/integration-LightCyberMagna.yml

```diff
@@ -494,4 +510,62 @@ script:
       default: true
       description: Host ID
     description: Get indicators for a specific detected host
+  - name: lcm-host-opened-ports
+    arguments:
+    - name: sort
+      description: How to sort the results
+    - name: host_id
+      required: true
+      description: Host id as presented in LightCyber Magna
+    description: Shows the used ports in this host
+  - name: lcm-host-suspicious-artifacts
+    arguments:
+    - name: sort
+      defaultValue: final_score
+    - name: host_id
+      required: true
+      default: true
+      description: Host id as presented in LightCyber Magna
+    description: Returns all the suspicious artifacts of this host
+  - name: lcm-host-processes
+    arguments:
+    - name: time
+    - name: sort
+      defaultValue: -score_e
+    - name: host_id
+      required: true
+      default: true
+      description: Host id as presented in LightCyber Magna
+    description: Returns all the running processes in the host
+  - name: lcm-host-loaded-modules
+    arguments:
+    - name: sort
+      defaultValue: final_score
+    - name: host_id
+      required: true
+      default: true
+      description: Host id as presented in LightCyber Magna
+    description: Returns all the modules of this host
+  - name: lcm-host-processes-internet-connections
+    arguments:
+    - name: time
+    - name: sort
+      defaultValue: process_sig
+    - name: src_host_id
+      required: true
+      default: true
+      description: Host id as presented in LightCyber Magna
+    description: Returns all the processes that established internet connection in
+      this host
+  - name: lcm-host-autoruns
+    arguments:
+    - name: sort
+      defaultValue: final_score
+    - name: host_id
+      required: true
+      default: true
+      description: Host id as presented in LightCyber Magna
+    description: Returns all installed autoruns programs in this host
   isfetch: true
+hidden: false
+releaseNotes: "Added the commands lcm-host-autoruns, lcm-host-processes-internet-connections, lcm-host-loaded-modules, lcm-host-processes, lcm-host-processes, lcm-host-suspicious-artifacts, lcm-host-opened-ports"
```

Comment:
- > `Added the following commands`
@gshriki what do you say ?

What about context & important to any of these commands ?